app/upload/views.py

```diff
@@ -200,9 +203,14 @@ def delete(r_id_type, r_id, filecode):
     return jsonify(response), 200
 
 
-@upload.route('/<request_id>', methods=['GET'])
-def get(request_id):
-    return jsonify({}), 200
+@upload.route('/<response_id>', methods=['GET'])
```

Comment:
Should I get rid of this? Are we ever going to need an `/upload/<response_id>` route that returns file information?